### DISCUSSION TITLE
Pool Size 에 대해

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#84] [General]
# Pool Size 에 대해

Pool Size에 대해 를 번역하였습니다. 다른 변경 사항이 없는 상태에서 커넥션 풀 크기만 줄였을 때 애플리케이션 응답 시간이 최대 100ms 에서 2ms 로 50배 이상 개선되었다는 Oracle 의 성능 측정 결과가 있었다. 실제로는 단일 코어는 한 번의 하나의 스레드만 실행할 수 있고, 운영체제가 컨텍스트를 전환할 때 해당 코어가 다른 스레드의 코…

https://vero.wiki/about-pool-sizing/

**Comment by somsom13** [comment]
인간 번역기네요^^ HikariCP 공부하고 있었는데 베로위키 염탐하고 갑니다~

**Reply by Cyma-s** [reply]
My Pleasure ^^ 
@>----------
